mobile/lib/firebase_options_dev.dart

```diff
@@ -0,0 +1,70 @@
+// File generated by FlutterFire CLI.
+// ignore_for_file: lines_longer_than_80_chars, avoid_classes_with_only_static_members
+import 'package:firebase_core/firebase_core.dart' show FirebaseOptions;
+import 'package:flutter/foundation.dart'
+    show defaultTargetPlatform, kIsWeb, TargetPlatform;
+
+/// Default [FirebaseOptions] for use with your Firebase apps.
+///
+/// Example:
+/// ```dart
+/// import 'firebase_options.dart';
+/// // ...
+/// await Firebase.initializeApp(
+///   options: DefaultFirebaseOptions.currentPlatform,
+/// );
+/// ```
+class DefaultFirebaseOptions {
+  static FirebaseOptions get currentPlatform {
+    if (kIsWeb) {
+      throw UnsupportedError(
+        'DefaultFirebaseOptions have not been configured for web - '
+        'you can reconfigure this by running the FlutterFire CLI again.',
+      );
+    }
+    switch (defaultTargetPlatform) {
+      case TargetPlatform.android:
+        return android;
+      case TargetPlatform.iOS:
+        return ios;
+      case TargetPlatform.macOS:
+        throw UnsupportedError(
+          'DefaultFirebaseOptions have not been configured for macos - '
+          'you can reconfigure this by running the FlutterFire CLI again.',
+        );
+      case TargetPlatform.windows:
+        throw UnsupportedError(
+          'DefaultFirebaseOptions have not been configured for windows - '
+          'you can reconfigure this by running the FlutterFire CLI again.',
+        );
+      default:
+        throw UnsupportedError(
+          'DefaultFirebaseOptions are not supported for this platform.',
+        );
+    }
+  }
+
+  static const FirebaseOptions android = FirebaseOptions(
+    apiKey: 'AIzaSyAcra-90bskaBWH8i-B_TR9EDeYQO36r_I',
+    appId: '1:394593516776:android:89d626c5cd0c10fa9e1d5d',
+    messagingSenderId: '394593516776',
+    projectId: 'airqo-dev',
+    databaseURL:
+        'https://console.firebase.google.com/u/1/project/airqo-dev/database',
+    storageBucket: 'airqo-dev.appspot.com',
+  );
+
+  static const FirebaseOptions ios = FirebaseOptions(
+    apiKey: 'AIzaSyAcra-90bskaBWH8i-B_TR9EDeYQO36r_I',
```

Comment:
Thanks @BenjaminSsempala 
I do understand that this reveal of keys is "okay" from a security perspective. Just wondering if it is somehow avoidable? Reference: https://medium.com/podiihq/keeping-secret-keys-out-of-version-control-in-flutter-bcd2b1eb9c1b

Reply:
Thank you @Baalmart 
This was also mentioned by @NoahNsimbe, I intend to push some commits to add the firebase options files to git ignore, plus a couple of other small changes we discussed with @Mnoble-19 and Noah.

Reply:
Alright @BenjaminSsempala ....what is the current status on this?
Is this something to address in a separate PR or you want to address it from this PR?

Please note that there is no right or wrong answer here. :)

Reply:
@Baalmart, Yes we resolved to add the firebase option files to git ignore and add them to secret manager. 
In this PR.